src/content/get-started/cloud-provider-guides/eks.mdx

```diff
@@ -7,6 +7,8 @@ pagination_prev: get-started/overview
 pagination_next: get-started/install-okteto-cli
 ---
 
+import variables from '../../variables.json';
```

Comment:
could we use the format that Hugues used for the [TiersList instead](https://github.com/okteto/docs/tree/main/src/theme/TiersList)? (seems easier to import to me)

Reply:
In the import? No if we want to support this with the versioned docs. Like I mentioned that's the only problem I see with this approach. The import has to be relative since it refers to a different json file in each version, so it has to be relative to the file that is importing it. The `TiersList` is a component that always lives in the "same place", independently of the version and can be referred in absolute terms. As far as I know there are no module "shorthand" for versioned folders, that would be a nice feature.

Reply:
Let's not spend more time on this. The new syntax is perfect for what we want.